ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -48,16 +56,28 @@ export function AssetPicker({ asset, onAssetChange }: AssetPickerProps) {
 
   const [showAssetPickerModal, setShowAssetPickerModal] = useState(false);
 
-  let image: string | undefined;
+  let primaryTokenImage: string | undefined;
 
   if (asset.type === AssetType.native) {
-    image = nativeCurrencyImageUrl;
+    primaryTokenImage = nativeCurrencyImageUrl;
```

Comment:
Nit: Is this name related to how `isTokenPrimary` is used in CurrencyInput? If not, I'd use a different name for clarity

Reply:
It's not related, but it's a different file and "primary" is a general term so I'd rather leave it honestly.